libff/algebra/curves/alt_bn128/alt_bn128_pairing.cpp

```diff
@@ -182,14 +186,14 @@ alt_bn128_Fq12 alt_bn128_final_exponentiation_last_chunk(const alt_bn128_Fq12 &e
       M = K * E              // = elt^(12*z^3 + 12*z^2 + 6*z)
       N = M * elt            // = elt^(12*z^3 + 12*z^2 + 6*z + 1)
       O = L.Frobenius_map(1) // = elt^(q*(12*z^3 + 6*z^2 + 4*z))
-      P = O * N              // = elt^(q*(12*z^3 + 6*z^2 + 4*z) * (12*z^3 + 12*z^2 + 6*z + 1))
-      Q = K.Frobenius_map(2) // = elt^(q^2 * (12*z^3 + 6*z^2 + 6*z))
-      R = Q * P              // = elt^(q^2 * (12*z^3 + 6*z^2 + 6*z) + q*(12*z^3 + 6*z^2 + 4*z) * (12*z^3 + 12*z^2 + 6*z + 1))
-      S = conj(elt)          // = elt^(-1)
-      T = S * L              // = elt^(12*z^3 + 6*z^2 + 4*z - 1)
-      U = T.Frobenius_map(3) // = elt^(q^3(12*z^3 + 6*z^2 + 4*z - 1))
-      V = U * R              // = elt^(q^3(12*z^3 + 6*z^2 + 4*z - 1) + q^2 * (12*z^3 + 6*z^2 + 6*z) + q*(12*z^3 + 6*z^2 + 4*z) * (12*z^3 + 12*z^2 + 6*z + 1))
-      result = V
+      P = O * N              // = elt^(q*(12*z^3 + 6*z^2 + 4*z) * (12*z^3 +
+      12*z^2 + 6*z + 1)) Q = K.Frobenius_map(2) // = elt^(q^2 * (12*z^3 + 6*z^2
```

Comment:
That would be great to keep the original format on this comment. Clang-format makes it hard to read now.

Reply:
Used `clang-format off` for this whole block.